compiler/damlc/daml-stdlib-src/DA/Experimental/Example.daml

```diff
@@ -0,0 +1,13 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module DA.Experimental.Example(
```

Comment:
This needs at least a `-- | HIDE` to hide it from the documentation but I’m not sure we need to commit this at all?

Reply:
I though that because it was not linked by `LibraryModules.daml`, it will not end up in the doc. 
I will add the flag.

Reply:
Maybe it is but that seems easy to change accidentally so regardless I think the `-- | HIDE` is worth adding for clarity.